elections/README.md

```diff
@@ -0,0 +1,116 @@
+can find the list of our current approvers and reviewers.
+
+Approvers and reviewers are also responsible for the Elections roadmap, 
+maintenance, and security.  As such, new approvers must be approved by
+the [Contributor Experience] chairs or by the Steering Committee.
+
+### Communications
+
+The elections subproject can be reached by the following mechanisms:
+
+* tagging area/elections in issues or PRs in the Kubernetes/Community repo
+* #sig-contribex channel in Kubernetes slack
+* The weekly [Contributor Experience] meeting
+
+### Documentation
+
+The subproject is responsible for making sure that all elections documentation
+is completed and kept up-to-date.  This includes:
+
+* Community documentation on the SC election
+* Election Officer documentation on how to run an SC election
+* Election messaging templates
+* Elekto documentation
+
+This documentation may include setting policy for some aspects of the Steering
+election.  In those cases, the subproject is responsible for getting the SC's
+approval on such items.
+
+### Software
+
+Elections in Kubernetes run on [Elekto].  The subproject is responsible for
+maintaining these by working together with [K8s-Infra] team.  This includes
+upgrades, migrations, assisting community members using the software, and handling
+security reports.  Should there be a reason to change software, the subproject
+will prepare recommendations for the Steering Committee to approve. It is also
+responsible for any necessary scripts, such as how to pull a voter list.
+
+### Recommending Election Officers
+
+The Elections Subproject will be responsible for finding and recommending 
+Election Officers to run the next Steering Committee election. The schedule 
+for that should be:
+
+* Early June: contact last year's EOs and determine who will be returning.
+* Mid-June: put out a call within SIG-Contribex to find out who is interested
+  in being an EO. Contact likely individuals 1-on-1
+* Early July: submit a recommended slate of EOs to the Steering Committee,
+  including alternates if possible.
+* Mid-July: SC approves the EOs.
+* Late July: EOs set the schedule for the election.
+
+Election Officers should be chosen from among regular, trusted contributors to the 
+Kubernetes project, with an eye towards employer, demographic, and geographic
+diversity.
+
+Election Officers are considered part of the Elections Subproject.
+
+### Minor Elections
+
+The subproject will assist Kubernetes teams/SIGs/WGs in preparing any internal
+elections they want to run.  This includes watching for election issues and PRs
+and either helping create the appropriate files or auditing them for the teams.
+The subproject will work with the Contributor Comms team to promote the elections
+as appropriate.
+
+Any named Kubernetes team (see [requesting an election]) may run an election 
+in elections.k8s.io.  As they cost nothing to run, and multiple elections can
```

Comment:
Is this statement true? I would expect there to be *some* cost associated with running an election, even if it is relatively trivial, since Elekto is running somewhere.

Maybe rewording this a bit to something like:
Since Elekto allows running multiple elections concurrently, the main limitation for running additional elections is the time of Contribex volunteers assist with the process.